data/content/agent_attributes.yaml

```diff
@@ -4,13 +4,13 @@ attributes:
   default_value: None
   required: true
   desc: |
-      The agent registration token from your organization's _Agents_ page, used only to register new agents. To get your token, log in to Buildkite, navigate to _Agents_, then select _Reveal Agent Token_.
+      The agent token associated with a cluster (and configurable through the cluster accessible from your organization's **Agents** page).
```

Comment:
Will change this to:

> The agent token associated with a cluster. The agent token is configurable through the cluster that's accessible from your organization's **Agents** page.

Reply:
Nice! One last thought, as I read your improved version -- is there always only one cluster available? The "the" (**the** cluster that's available) makes me think this.

If not, is it more:

> The agent token associated with a cluster. The agent token is configurable from your organization's Agents page, in the clusters section.

Or something. That's not quite right, but hopefully helps to illustrate my point. I haven't looked at the clusters UI changes or agent<>cluster relationships we shipped lately, please use your more complete info on this one!

Reply:
It's possible for any number (afaik) of clusters to be configured within a given Buildkite org. However, agent tokens are now only configurable through (and quarantined to) individual clusters. Therefore, the agent tokens configured in one cluster cannot be used with another cluster (which is by design).

The way to access / create new / update / revoke a cluster's agent tokens (through the UI) is by clicking/selecting **Agents** (from global nav) > the cluster in question (e.g. **Default cluster**) > **Agent tokens**.

I suppose it might be worthwhile including a link to the [Agent tokens](https://buildkite.com/docs/agent/v3/tokens) page of our docs, which explains this in much more detail, but at this stage, I'd like to avoid setting a docs linking precedent from tabular content like this. If we do get customer feedback for this, then I might start doing this.